server/http_handler.go

```diff
@@ -273,6 +274,25 @@ func (t *tikvHandlerTool) handleMvccGetByHex(params map[string]string) (interfac
 	return t.getMvccByEncodedKey(encodedKey)
 }
 
+func (t *tikvHandlerTool) GetAllHistory() ([]*model.Job, error) {
```

Comment:
Is there a way to not expose this function for test? I think the data in the test are known